packages/react-native-client/src/components/VideoRendererView.tsx

```diff
@@ -14,6 +14,10 @@ export type VideoRendererProps = {
    * @default `FILL`
    */
   videoLayout?: VideoLayout;
+  /**
+   * Whether the video should be rendered if it's off screen or hidden.
+   */
+  shouldRenderIfNotVisible?: boolean;
```

Comment:
I would consider changing name of this props, to something more like:
`skipRenderOutsideVisibleArea`
This way:
- there is no negation in prop name (it is often confusing)
- props will be opt in (as you will have to set it to true, for this functionality to be used)